tests/sentry/manager/tests.py

```diff
@@ -14,37 +15,48 @@ def test_valid_only_message(self):
         self.assertEquals(event.message, "foo")
         self.assertEquals(event.project_id, 1)
 
+    @pytest.mark.skipIf(
+        lambda x: settings.SENTRY_TAGSTORE
+        in [
+            "sentry.tagstore.snuba.SnubaCompatibilityTagStorage",
+            "sentry.tagstore.snuba.SnubaTagStorage",
+        ]
+    )
     def test_add_tags(self):
-        event = Group.objects.from_kwargs(1, message="rrr")
-        group = event.group
-        environment = self.create_environment()
-
-        with self.tasks():
-            Group.objects.add_tags(
-                group, environment, tags=[("foo", "bar"), ("foo", "baz"), ("biz", "boz")]
+        if settings.SENTRY_TAGSTORE in [
```

Comment:
What's the goal of this test if we are getting rid of the legacy tagstore ?

Reply:
deleted